cmd/cloudx/relationtuples/relationtuples.go

```diff
@@ -68,15 +68,20 @@ func forwardConnectionInfo(cmd *cobra.Command) {
 			return err
 		}
 
-		key, err := h.CreateAPIKey(ctx, project.Id, "keto-temp-"+randx.MustString(8, randx.AlphaNum))
+		key, err := h.CreateProjectAPIKey(ctx, project.Id, "keto-temp-"+randx.MustString(8, randx.AlphaNum))
 		if err != nil {
 			return err
 		}
-		defer func() { _ = h.DeleteAPIKey(ctx, project.Id, key.Id) }()
+		defer func() { _ = h.DeleteProjectAPIKey(ctx, project.Id, key.Id) }()
```

Comment:
Keto should also use OAuth2. I can do that in a follow-up.

Reply:
True, let's do that in a follow-up.